src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # BinSkim Release History
 
+## Unreleased
+
+* FEATURE: Add new PE CV_CFL_LANG language code for ALIASOBJ and Rust. [530](https://github.com/microsoft/binskim/pull/530)
```

Comment:
>FEATURE [](http://example.com/codeflow?start=2&length=7)

I don't think FEATURE is buying us much in this document. This release note item doesn't describe the additional value the change is providing.

Reply:
agree, changed to the other one BUGFIX